prime-router/settings/organizations.yml

```diff
@@ -975,6 +975,36 @@
         filePath: ./upload
         credentialName: DEFAULT-SFTP
     # THIS RECEIVER IS NOT USED IN ANY TESTS AND CAN BE UPDATED TO YOUR HEARTS CONTENT
+    - name: ny-development
```

Comment:
Typically we leave these local and don't merge to master as it has a tendency to get really cluttered after awhile.

Reply:
Is there another way of doing it so that whoever is reviewing the PR can test this? Do you create a separate file specific for each receiver?

Reply:
I pushed a separate file with only the NY receiver settings and updated the PR test steps to use it.

Reply:
Yep, I' usually just include the yml in the PR test steps so people can copy paste.